features/filiatie.feature

```diff
@@ -0,0 +1,618 @@
+    | gebeurtenis                               | status oude kadastraal onroerende zaak/zaken | oude kadastraal onroerende zaak/zaken raadpleegbaar |
+    | splitsen van perceel in nieuwe percelen   | vervallen                                    | ja                                                  |
+    | samenvoegen van percelen                  | vervallen                                    | ja                                                  |
+    | splitsen perceel in appartementsrechten   | actueel                                      | ja                                                  |
+    | ondersplitsing van appartementsrecht      | sluimerend                                   | ja                                                  |
+    | wijzigingssplitsing van appartementsrecht | vervallen                                    | nee                                                 |
+
+
+    Naast de filiatie worden relaties naar de grondpercelen waaruit het appartementsrecht is ontstaan of naar de appartementsrechten horend bij het perceel geleverd, zodat direct de bijbehorende grondpercelen dan wel bijbehorende appartementsrechten kunnen worden opgehaald, zonder eerst via eventuele sluimerende appartementsrechten de filiatie af te hoeven lopen.
+
+    #TODO: invullen onroerende zaak identificaties
+
+    Rule: Na splitsing van een perceel wordt filiatie opgenomen bij het vervallen perceel en bij de nieuwe percelen
+        Scenario: Opvragen van een vervallen perceel na splitsing
+            Gegeven Perceel met identificatie "P1" is gesplitst naar percelen "P2" en "P3"
+            En Perceel met identificatie "P1" is niet ontstaan uit splitsing of vereniging van een ander perceel
+            En Perceel met identificatie "P2" is niet verder gesplitst of verenigd
+            En Perceel met identificatie "P3" is niet verder gesplitst of verenigd
+            Als "/kadastraalonroerendezaken/P1" wordt gevraagd
+            Dan bevat het antwoord "isOvergegaanIn" met waarde:    
+                """
+                [
+                    {
+                        "aard": {
+                            "code": "20",
+                            "waarde": "Splitsen perceel"
+                        },
+                        "kadastraalOnroerendeZaak": {
+                            "identificatie": "P2"
+                        }
+                    },
+                    {
+                        "aard": {
+                            "code": "20",
+                            "waarde": "Splitsen perceel"
+                        },
+                        "kadastraalOnroerendeZaak": {
+                            "identificatie": "P3"
+                        }
+                    }
+                ]
+                """
+            En bevat het antwoord "indicatieVervallen": true
+            En bevat het antwoord property "einddatum" met een waarde
+            En is de kadastraal onroerende zaak niet sluimerend
```

Comment:
ik weet niet hoe dit moet worden gelezen. Heeft deze KOZ wel indicatieSluimerend en is de waarde false en wordt hij niet getoond of is de waarde null en wordt hij niet getoond

Reply:
we hebben in deze release geen property indicatieSluimerend. Dus dit is puur ter illustratie om uit te leggen in welke gevallen een zaak wel of niet sluimerend is/genoemd wordt. Dit heeft in deze release dus nog geen consequenties voor wat de API levert.